src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/search/PersonMatchService.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.hmpps.personrecord.service.search
+
+import kotlinx.coroutines.runBlocking
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.personrecord.client.PersonMatchClient
+import uk.gov.justice.digital.hmpps.personrecord.client.model.match.PersonMatchScore
+import uk.gov.justice.digital.hmpps.personrecord.jpa.entity.PersonEntity
+import uk.gov.justice.digital.hmpps.personrecord.service.EventKeys
+import uk.gov.justice.digital.hmpps.personrecord.service.RetryExecutor
+import uk.gov.justice.digital.hmpps.personrecord.service.TelemetryService
+import uk.gov.justice.digital.hmpps.personrecord.service.type.TelemetryEventType.MATCH_CALL_FAILED
+
+@Component
+class PersonMatchService(
+  private val personMatchClient: PersonMatchClient,
+  private val telemetryService: TelemetryService,
+  private val retryExecutor: RetryExecutor,
+) {
+
+  fun getScores(personEntity: PersonEntity): List<PersonMatchScore> = runBlocking {
+    try {
+      return@runBlocking retryExecutor.runWithRetryHTTP { personMatchClient.getPersonScores(personEntity.matchId.toString()) }
+    } catch (exception: Exception) {
+      telemetryService.trackEvent(
```

Comment:
Do we really need this telemetry? It will appear in AppInsights as a 500 (or whatever) and be easy to retrieve?

Reply:
I know it's a convention we have adopted, but I feel it is unnecessary to have a try/catch simply for this purpose